apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/Trace/AbstractTraceHeaderPropagatingHandlerTest.cs

```diff
@@ -123,7 +123,19 @@ public FakeDelegatingHandler(TraceHeaderContext context = null, bool throwExcept
                 }
                 return Task.FromResult(new HttpResponseMessage());
             }
+        }
+
+        private class TraceHeaderPropagatingHandler : AbstractTraceHeaderPropagatingHandler
```

Comment:
FakeTraceHeaderPropagatingHandler? It's confusing to give this the same name as the prod class.